### PR TITLE
Permission Constant

### PR DESCRIPTION
**Pull request by knockknockyoo**

### CODE REVIEW

File: constants/user-permission-constants.js [hunk 1]
```diff
@@ -1,41 +1,45 @@
...
-        SUBMISSION: {
-            SUBMIT: "Submit",
-            CANCEL: "Cancel",
-            RELEASE: "Release",
-            COMPLETE: "Complete",
-            DELETE: "Delete",
-            WITHDRAW: "Withdraw",
-            REMIND_EXPIRE: "Remind Expire"
+        DATA_SUBMISSION: {
+            SUBMIT: "data_submission:request_access",
```
Comment (n2iw) [inline constants/user-permission-constants.js]: This should be **data_submission:submitted**

File: constants/user-permission-constants.js [hunk 2]
```diff
@@ -1,41 +1,45 @@
...
+            CANCEL: "data_submission:cancelled",
+            RELEASE: "data_submission:released",
+            COMPLETE: "data_submission:complete",
```
Comment (n2iw) [inline constants/user-permission-constants.js]: Should be data_submission:completed

**Review by n2iw — CHANGES_REQUESTED** [review]

File: constants/user-permission-constants.js (hunk 1 above)
Comment (knockknockyoo) [reply constants/user-permission-constants.js]: fixed it.

File: constants/user-permission-constants.js (hunk 2 above)
Comment (knockknockyoo) [reply constants/user-permission-constants.js]: fixed it.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**